src/yield/yielder.cairo

```diff
@@ -11,6 +11,7 @@ from openzeppelin.access.ownable.library import Ownable
 from openzeppelin.upgrades.library import Proxy
 
 // Local dependencies
+from src.offset.library import CarbonableOffseter
```

Comment:
General comment for this file :
The following function are only data know by the `Offseter` and are only getter (without Yielder implication or data):
- `getTotalDeposited`
- `getTotalClaimed`
- `getTotalClaimable`
- `getClaimableOf`
- `getClaimedOf`

So, this functions should not be only exposed by the `Offseter.cairo` ? instead to find it into `Yielder.cairo`.

Reply:
To clarify, I'll rename Claimable by Absorption in the Yielder.